qlib/backtest/__init__.py

```diff
@@ -199,12 +217,29 @@ def get_strategy_executor(
     )
 
     exchange_kwargs = copy.copy(exchange_kwargs)
+    # derive freq from executor config if not explicitly provided
+    if "freq" not in exchange_kwargs:
+        try:
+            if isinstance(executor, dict):
+                tps = executor.get("kwargs", {}).get("time_per_step")
+                if isinstance(tps, str) and tps:
+                    exchange_kwargs["freq"] = tps
+        except Exception:
```

Comment:
Avoid unclear exception.

Reply:
We have a lot of unclear exception in this PR.
It would be great if we can change it into conerete exception with comments.